PyTorch/vLLM_Tutorials/Deploying_vLLM/template_vllm_server_fp8.sh

```diff
@@ -30,17 +34,34 @@ if [ -n "$PT_HPU_RECIPE_CACHE_CONFIG" ]; then # Checks if using recipe cache
 fi
 
 cd /root/scripts
-## Start vLLM FP8 server  
-QUANT_CONFIG=./measurement/${${q}MODEL_BASE}/maxabs_quant_$gnum.json vllm serve $MODEL \
-        --quantization=${${q}QUANTIZATION} \
-        --kv_cache_dtype=${${q}KV_CACHE_DTYPE} \
-        --tensor-parallel-size=$TENSOR_PARALLEL_SIZE \
-        --max-model-len=$MAX_MODEL_LEN \
-        --dtype bfloat16 \
-        --gpu-memory-util $GPU_MEM_UTILIZATION \
-        --use-padding-aware-scheduling \
-        --max-num-seqs $MAX_NUM_SEQS \
-        --max-num-prefill-seqs $MAX_NUM_PREFILL_SEQS \
-        --num_scheduler_steps 1 \
-        --weights-load-device cpu \
-        --disable-log-requests ${${q}EXTRA_ARGS}
+## Start vLLM FP8 server
+if [ "${${q}MODEL_FAMILY}" = "mistralai" ]; then
```

Comment:
Is the difference only is kv_cache_dtype? Then handle the param based on model not the entire cmdline.